Formula/c/crossplane.rb

```diff
@@ -0,0 +1,18 @@
+class Crossplane < Formula
+  desc "Build control planes without needing to write code"
+  homepage "https://github.com/crossplane/crossplane"
+  url "https://github.com/crossplane/crossplane/archive/refs/tags/v1.17.1.tar.gz"
+  sha256 "44e8c94ffa41174f12e25ff60a9c7fd30cfd404b07b2332ec04070a27c2d2e74"
+  license "Apache-2.0"
+  head "https://github.com/crossplane/crossplane.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w -X=github.com/crossplane/crossplane/internal/version.version=v#{version}", output: bin/"crossplane"), "./cmd/crank"
```

Comment:
why just build crank, but not crossplane as well?

Reply:
I don't quite get what's the use of crossplane binary for normal users. Following https://docs.crossplane.io/latest/cli/ the shell script referenced installs only the `crank` binary

Reply:
Crank is the crossplane cli, crossplane is the actual crossplane binary, so in this context only crank makes sense indeed.

Reply:
I think the `crossplane` binary runs in an Init container to install some crossplane CRDs to a cluster where someone is installing Crossplane via Helm. It does not seems to be intended for running on local machines.

Reply:
Both as init container and main container. I confirm it's not meant to be run locally.

Reply:
sounds good then, just checking 👍

Reply:
(related, would it make more sense to have the formula as crank, rather than crossplane then?)

Reply:
Crank is just an internal name, the cli should be called crossplane, so crossplane is correct 👍

Reply:
So `brew install crossplane` will mean that there is no `crossplane` binary, just a `crank` one?

Reply:
`brew install crossplane` will install the `crank` binary as `crossplane`.